<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QMainWindow" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètrage pour afficheur Pixelix</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="urlLabel">
         <property name="text">
          <string>URL Pixelix :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="urlEdit">
         <property name="text">
          <string>http://172.21.28.41</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="couleurButton">
         <property name="text">
          <string>Couleur du texte</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ScrollButton">
         <property name="text">
          <string>Scrolling</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="send_txt">
       <property name="text">
        <string>Hello World</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="envoyerButton">
       <property name="text">
        <string>Envoyer texte</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="configLabel">
       <property name="text">
        <string>Brightness :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="telechargementButton">
       <property name="text">
        <string>Télécharger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QLabel" name="brightnessLabel">
       <property name="text">
        <string>Brightness :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="brightnessprogressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="TrameLabel">
         <property name="text">
          <string> Trame :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TrameEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="EnvoyerButtonTrame">
         <property name="text">
          <string>Envoyer trame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkAnim">
         <property name="text">
          <string>8x8 Animé</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>telechargementButton</sender>
   <signal>clicked()</signal>
   <receiver>fenetre</receiver>
   <slot>telecharger_brightness()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>envoyerButton</sender>
   <signal>clicked()</signal>
   <receiver>fenetre</receiver>
   <slot>envoyer_txt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>couleurButton</sender>
   <signal>clicked()</signal>
   <receiver>fenetre</receiver>
   <slot>choixcouleur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>fenetre</receiver>
   <slot>changer_url()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EnvoyerButtonTrame</sender>
   <signal>clicked()</signal>
   <receiver>fenetre</receiver>
   <slot>envoyer_Trame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>telecharger_brightness()</slot>
  <slot>envoyer_txt()</slot>
  <slot>choixcouleur()</slot>
  <slot>changer_url()</slot>
  <slot>envoyer_Trame()</slot>
  <slot>setTrame()</slot>
 </slots>
</ui>
